README.md

```diff
@@ -93,7 +91,7 @@ Links to other lists of packages are in **bold**.
 
 ### Data processing
 
-   * [awkward-array](https://awkward-array.org): Support "Jagged" arrays (like TTree's) easily in Python. Also supports Numba.
+   * [Awkward Array](https://github.com/scikit-hep/awkward-1.0): Manipulate JSON-like data with NumPy-like idioms.
```

Comment:
Was this intentional? I think `awkward-array.org` is a better (more stable) thing to link to?

Reply:
Adding the new catchphrase was intentional, reverting to the repo was not. In the conflict resolution, I don't think I was presented with `https://awkward-array.org` as an option, but you're right that it's a better choice than the GitHub README.